inc/taxonomies-functions.php

```diff
@@ -2243,3 +2248,37 @@ function taxopress_show_all_cpt_in_archive_result($request_tax){
 
             return $status;
 }
+
+/* Show taxonomy filter on post list */
+function taxopress_filter_dropdown( $taxonomy ) {
+	wp_dropdown_categories(
+		array(
+			'show_option_all' => sprintf( __( 'All %s', 'admin-taxonomy-filter' ), $taxonomy->label ),
+			'orderby'         => 'name',
+			'order'           => 'ASC',
+			'hide_empty'      => false,
+			'hide_if_empty'   => true,
+			'selected'        => filter_input( INPUT_GET, $taxonomy->query_var, FILTER_SANITIZE_STRING ),
+			'hierarchical'    => true,
+			'name'            => $taxonomy->query_var,
+			'taxonomy'        => $taxonomy->name,
+			'value_field'     => 'slug',
+		)
+	);
+}
+function taxopress_get_dropdown(){
+	if ( is_admin() ) {
+		$taxonomies        = taxopress_get_taxonomy_data();
+		$all_taxonomies    = get_all_taxopress_taxonomies();
+		foreach ( $all_taxonomies as $taxonomy ) {
+			$taxonomy_name = $taxonomy->name;
+			$current       = $taxonomies[ $taxonomy_name ];
```

Comment:
While testing, i keep getting `PHP Notice:  Undefined index: category in /sites/publishpress/wp-content/plugins/TaxoPress-filterforpostsscreen/inc/taxonomies-functions.php on line 2275` warning instead of getting the dropdown.

Can you look into this and confirm everything is working?